tools/testing/target_determination/heuristics/interface.py

```diff
@@ -458,3 +458,13 @@ def name(self) -> str:
 
     def __str__(self) -> str:
         return self.name
+
+    @abstractmethod
+    def get_test_ratings(self, tests: List[str]) -> Dict[str, float]:
```

Comment:
Since rating really means confidence score here, how about a rename? 

```suggestion
    def get_prediction_confidence(self, tests: List[str]) -> Dict[str, float]:
```